skimage/color/colorconv.py

```diff
@@ -844,10 +844,10 @@ def gray2rgb(image, alpha=None):
             is_rgb = True
 
     if is_rgb:
-        if alpha == False:
+        if alpha is False:
             image = image[..., :3]
 
-        elif alpha == True and not is_alpha:
+        elif alpha is True and is_alpha is False:
```

Comment:
Travis' flake8 says something different. Maybe more recent flaske8 versions are ok with the use of `not`